site_tmpl/position_list.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base_admin.html' %}
+{% load bootstrap3 %}
+
+{% block content %}
+<h1>Open positions</h1>
+<p>These are positions that the executive board is seeking applications for.
```

Comment:
Not all positions should require applications.

Reply:
Resolved in b926a9f